lib/components/Search.dart

```diff
@@ -3,16 +3,25 @@ import 'package:veggie_seasons_v2/styles/styles.dart';
 import 'package:flutter/material.dart';
 
 class Search extends StatelessWidget {
-  const Search(
-      {super.key,
-      required this.editingController,
-      required this.filterSearchResults});
+  const Search({
+    super.key,
+    required this.editingController,
+    required this.filterSearchResults,
+  });
 
   final TextEditingController editingController;
   final Function filterSearchResults;
 
   @override
   Widget build(BuildContext context) {
+    final OutlineInputBorder border = OutlineInputBorder(
```

Comment:
Consolidated since it was being duped, more concerned about possible bug mentioned below though.

Reply:
Can this be reused in the Styles.dart file? Admittedly I still confuse OutlineInputBorder and BoxDecoration. It _feels_ like they should be the same thing, but I know it's not.

Reply:
Oh for sure. You can put whatever you like in the style file. I noticed there were some box decorations there, but then box decorations elsewhere in the app. I did not go so far as to refactor everything to put consistent box decorations in the style file for reuse throughout the app. That could be a nice cleanup though!